website/docs/dapp/simple-lock.mdx

```diff
@@ -0,0 +1,376 @@
+---
+id: simple-lock
+title: Simple Lock Example
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import TutorialHeader from "@components/TutorialHeader";
+import { LOCKOVERVIEW } from "./TutorialOverview.tsx";
+import SwitchToTestnet from "./_SwitchToTestnet.mdx";
+import StartDevnet from "./_StartDevnet.mdx";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import ImgContainer from "@components/ImgContainer";
+import CodeTabs from "./_CodeTabs.mdx";
+
+# Simple Lock Example
```

Comment:
Can we rename this to "Build a Lock" to make it more consistent with other titles?

Reply:
How about `Build a Simple Lock`?